applications/peripheral_sensor_node/README.rst

```diff
@@ -0,0 +1,260 @@
+
+Configuration
+*************
+
+The nRF Peripheral Sensor node application is modular and event-driven.
+You can enable and configure the modules separately for the selected board and build type.
+See the :ref:`lib_caf` documentation for more information.
+
+Configuration options
+=====================
+
+Check and configure the following Kconfig options:
+
+.. _CONFIG_APP_CONTROL_BURST_INTERVAL_S:
+
+CONFIG_APP_CONTROL_BURST_INTERVAL_S
+   The application configuration sets the time interval in seconds for burst data acquisitions from sensors.
+   If sensor data burst is not triggered by :c:struct:`wake_up_event` from IMU, burst is performed at each interval.
+
+.. _CONFIG_APP_CONTROL_BURST_MEAS_CNT:
+
+CONFIG_APP_CONTROL_BURST_MEAS_CNT
+   The application configuration specifies the number of measurements a single burst acquires before going to sleep.
+   The default value is ``10``.
+
+.. _CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS:
+
+CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS
+   The application configuration specifies the time interval between measurements for a single burst.
+   The default value is 1000 ms.
+
+Configuration files
+===================
+
+The nRF Peripheral Sensor node application uses both the :ref:`standard configuration files common to Zephyr and the |NCS| <app_build_system>` and the application-specific :file:`_def` files, which contain configuration arrays for the application modules.
+The :file:`_def` files are used by the nRF Peripheral Sensor node application modules and :ref:`lib_caf` modules.
+
+Configuration files are defined in the :file:`applications/peripheral_sensor_node/configuration/nrf54l15dk_nrf54l15_cpuapp` directory.
+
+Advertising configuration
+-------------------------
+
+If a given build type enables Bluetooth, the :ref:`caf_ble_adv` is used to control the Bluetooth advertising.
+This CAF module relies on :ref:`bt_le_adv_prov_readme` to manage advertising data and to scan response data.
+The nRF Peripheral Sensor node application configures the data providers in the :file:`src/util/Kconfig` file.
+By default, the application enables a set of data providers available in the |NCS|.
+If the Noridc UART Service (NUS) is enabled in the configuration, and the Bluetooth local identity in use has no bond, the application also adds a custom provider that appends UUID128 of NUS to the scan response data.
+
+Shield Configuration
+====================
+
+To ensure that the measurement is triggered by an interrupt from the ADXL362 sensor, you need to set a jumper on the shield board.
+Place a jumper on header **P3** to connect the ``IRQ_U3`` signal.
+This configuration allows the application to respond to interrupts generated by the sensor.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`applications/peripheral_sensor_node`
```

Comment:
This will fix the doc build (sorry, my mistake, the include was different than usual).
```suggestion
.. |application path| replace:: :file:`applications/peripheral_sensor_node`
```